### PR TITLE
adds flye 2.9.2

### PR DESCRIPTION
**Pull request by kapsakcj**
added dockerfile and README.md for flye 2.9.2. Also added to main README.md

Dockerfile is almost the same as the v2.9.1 dockerfile.

Changes from previous 2.9.1 dockerfile:

- upgraded base image to `ubuntu:focal` from bionic
- changed apt-get install steps to `python3-*` since `python-*` packages do not exist for ubuntu:focal apt registries
- added apt-get install `ca-certificates` `procps`
- added myself as a maintainer
- adjusted flye install steps to use `python3` instead of `python`
- added test at the end including test nanopore data from bactopia/bactopia-tests github repo. Test shouldn't take longer than 5-7 minutes to run. Can comment out if it ends up taking a while on GHActions

Pull Request (PR) checklist:
- [x] Include a description of what is in this pull request in this message.
- [x] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
- [x] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
   - [x] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
- [x] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
   - [x] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
- [x] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18) 
- [x] Main [README.md](https://github.com/StaPH-B/docker-builds/blob/master/README.md) has been updated to include the tool and/or version of the dockerfile(s) in this PR
- [x] [Program_Licenses.md](https://github.com/StaPH-B/docker-builds/blob/master/Program_Licenses.md) contains the tool(s) used in this PR and has been updated for any missing

### CODE REVIEW

**Comment by kapsakcj** [comment]
My suggestion is to create a new docker image tag `staphb/flye:2.9.2` and overwrite `:latest` tags on both dockerhub and quay

**Comment by erinyoung** [comment]
You beat me to this. Thank you!

File: flye/2.9.2/README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Main tool : [flye](https://github.com/fenderglass/Flye) 2.9.2
+
+Full documentation: [https://github.com/fenderglass/Flye/blob/flye/docs/USAGE.md](https://github.com/fenderglass/Flye/blob/flye/docs/USAGE.md)
+
+> Flye is a de novo assembler for single-molecule sequencing reads, such as those produced by PacBio and Oxford Nanopore Technologies.
+
+Additional included tools:
+
+- [minimap2](https://github.com/lh3/minimap2) 2.24-r1122
```
Comment (erinyoung) [inline flye/2.9.2/README.md:11]: For future self : minimap2 is included in the flye install

**Review by erinyoung — APPROVED** [review]
I don't have any recommendations or changes to suggest.

I'm going to
1. merge this PR
2. push this to dockerhub and quay using the '2.9.2' tag
3. overwrite the 'latest' flag

**Pull request merged**